cpp/src/compaction/stream_compaction_ops.cu

```diff
@@ -149,185 +87,146 @@ struct bit_or: public thrust::unary_function<thrust::tuple<gdf_valid_type,gdf_va
 		return thrust::get<0>(x) | thrust::get<1>(x);
 	}
 };
- 
-
-typedef thrust::transform_iterator<modulus_bit_width, thrust::counting_iterator<gdf_size_type> > bit_position_iterator;
-
 
-template<typename stencil_type>
-struct is_stencil_true
+struct is_bit_set
```

Comment:
All of these functors should go into an anonymous namespace since they're only used locally.

Reply:
Suggested change committed.